otx/algorithms/classification/tasks/inference.py

```diff
@@ -352,7 +348,6 @@ def _init_recipe(self):
         # FIXME[Soobee] : if train type is not in cfg, it raises an error in default INCREMENTAL mode.
         # During semi-implementation, this line should be fixed to -> self._recipe_cfg.train_type = train_type
         self._recipe_cfg.train_type = train_type.name
-        patch_data_pipeline(self._recipe_cfg, self.data_pipeline_path)
```

Comment:
Could you let me know why you removed this line?
Is this relevant to this PR? OR you just applied some changes from other PRs?

Reply:
This change is because of error, so I followed up develop PR. Sorry for missed the PR number, I'll update when I check it

Reply:
Could you go over this?

Reply:
I missed out while following #1431, sorry for the confusion.